src/views/List.jsx

```diff
@@ -23,29 +24,40 @@ export function List({ data }) {
 	// TODO - Use navigation similar to in Home.jsx to navigate to home page if no token or token is null
 	return (
 		<>
-			<form onSubmit={handleSubmit}>
-				<label htmlFor="searchTerm">Filter list items</label>
-				<input
-					type="text"
-					value={searchTerm}
-					id="searchTerm"
-					placeholder="Start typing here..."
-					onChange={onSearch}
-				/>
-				{!!searchTerm && (
-					<button onClick={onSearchReset} type="reset">
-						Reset
-					</button>
-				)}
-			</form>
-			{!!filteredData.length || !searchTerm ? (
-				<ul>
-					{filteredData.map((item, i) => (
-						<ListItem key={item.name + i} {...item} />
-					))}
-				</ul>
+			{!data.length ? (
```

Comment:
Looks good! Nice and clean. This is totally optional, and I wonder if for readability, it makes sense to be consistent about our truthy/falsy operators. Since we are using `!!` in the subsequent code block, we could use it here as well and flip the returns of the ternary.

Reply:
@wheninseattle Thanks for that suggestion. I'd like to see if @mariahschock weighs in, as well as maybe get input from @veeteeran (who I believe was who originally made the recommendation to use !!). Admittedly, I did not keep up with that thread last week, and need to catch up to speed on why it was an improvement in that case. I don't mind changing it; I just want to be sure I understand the underlying reason for the change (beyond consistency/readability...personally I actually find it easier to read the other way, but that's probably just because of my lack of understanding). To be continued...

Reply:
@wheninseattle and @tmflower I agree with Tina that it's easier for me to read the way it is written. That being said, I'm not married to keeping it how it currently is and do agree that it may be beneficial to keep our truthy/falsy operators consistent across our application.

Reply:
@mariahschock and @tmflower, if you find it more intuitive/readable as is, then I think it makes sense to leave it! Cool to hear your thought process!